### PR TITLE
feat(i18n): add i18n support 

### PR DESCRIPTION
**Pull request by MrTimscampi**
* Add support for translation through vue-i18n/nuxt-i18n.
* Translate the existing messages into both English and French.
* Add a language switcher for quick switching (Probably temporary, but it helps for now)
* Add i18n-ally settings to VSCode workspace

image

### CODE REVIEW

File: components/LoginForm.vue [hunk 1]
```diff
@@ -84,25 +84,26 @@ export default Vue.extend({
...
       } catch (error) {
-        let errorMessage = 'Unexpected Error';
+        let errorMessage = this.$t('unexpectedError');
 
         if (!error.response) {
-          errorMessage = 'Server Not Found';
+          errorMessage = this.$t('serverNotFound');
         } else if (error.response.status === 500) {
-          errorMessage = 'Incorrect Password';
+          errorMessage = this.$t('incorrectPassword');
```
Comment (Camc314) [inline components/LoginForm.vue:92]: this could actually be incorrect username/password

Comment (MrTimscampi) [reply components/LoginForm.vue:92]: Hm, then the message definitely needs changing

File: locales/en-US.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "badRequest": "Bad Request. Try Again",
+  "home": "Home",
+  "incorrectPassword": "Incorrect Password",
```
Comment (Camc314) [inline locales/en-US.json:4]: same here. probably want to rename to `Incorrect Username or Password`

File: components/LoginForm.vue (hunk 1 above)
Comment (Camc314) [reply components/LoginForm.vue:92]: rename to `Incorrect Username or Password`

**Review by Camc314 — APPROVED** [review]

**Pull request merged**